src/Plugin/os2web/NemloginAuthProvider/OpenIDConnect.php

```diff
@@ -394,4 +419,48 @@ public function log($level, $message, array $context = []): void {
     }
   }
 
+  /**
+   * Get string of claim data.
+   */
+  private function getClaimDataString(): string {
+    $claimKeys = $this->getClaimKeys();
+
+    $claims = [];
+
+    foreach ($claimKeys as $claimKey) {
+      $claimValue = $this->fetchValue($claimKey);
+      if (!empty($claimValue)) {
+        $claims[] = $claimKey . ': ' . $this->fetchValue($claimKey);
```

Comment:
Could it be helpful to log the empty/missing claims? Maybe that more for debugging than audit.

Reply:
Handled in 704e7ff4b33ad5a8a3d388a9731102a3d1236465.